l10n.toml

```diff
@@ -2,51 +2,92 @@
 basepath = "."
 
 locales = [
+    "an",
+    "ar",
+    "ast",
+    "az",
+    "be",
     "bg",
+    "bn",
+    "br",
+    "bs",
     "ca",
+    "cak",
     "co",
     "cs",
     "cy",
     "da",
     "de",
     "dsb",
+    "el",
+    "en-CA",
     "en-GB",
+    "eo",
     "es",
     "es-AR",
+    "es-CL",
     "es-ES",
     "et",
     "eu",
     "fa",
+    "ff",
     "fi",
     "fr",
     "fy-NL",
+    "ga-IE",
     "gd",
+    "gn",
     "gu-IN",
     "he",
+    "hi-IN",
+    "hr",
     "hsb",
     "hu",
+    "hy-AM",
     "id",
+    "is",
     "it",
     "ja",
+    "ka",
     "kab",
+    "kk",
+    "kn",
     "ko",
+    "lij",
     "lo",
+    "lt",
     "ml",
+    "mr",
+    "ms",
+    "my",
     "nb-NO",
+    "ne-NP",
     "nl",
     "nn-NO",
     "nv",
+    "oc",
     "pa-IN",
     "pl",
     "pt-BR",
+    "pt-PT",
+    "rm",
+    "ro",
     "ru",
     "sk",
+    "sl",
+    "sq",
+    "sr",
     "su",
     "sv-SE",
-    "tr",
```

Comment:
@Pike Is the removal here intended? Do we need to get rid of strings in `values-tr` folders?